pkg/port/port.go

```diff
@@ -100,7 +100,9 @@ func GetListeningConnections(ctx context.Context, execClient exec.Client, podNam
 func GetConnections(ctx context.Context, execClient exec.Client, podName string, containerName string, statePredicate func(state int) bool) ([]Connection, error) {
 	cmd := []string{
 		remotecmd.ShellExecutable, "-c",
-		"cat /proc/net/tcp /proc/net/udp /proc/net/tcp6 /proc/net/udp6",
+		// /proc/net/{tc,ud}p6 files might be missing if IPv6 is disabled in the host networking stack.
+		// Actually /proc/net/{tc,ud}p* files might be totally missing if network stats are disabled.
+		"for f in tcp udp tcp6 udp6; do cat /proc/net/$f || true; done",
```

Comment:
You could also keep the previous command, which will display the files found and display an error for not found ones.

Reply:
Yes, we can do that. I didn't want to stop returning the content of the subsequent files if any file in the middle does not exist, but that seems to be already the case here.